lib/rucio/db/sqla/migrate_repo/versions/22d887e4ec0a_create_sources_table.py

```diff
@@ -54,7 +56,7 @@ def upgrade():
 
         create_primary_key('SOURCES_PK', 'sources', ['request_id', 'rse_id', 'scope', 'name'])
         create_foreign_key('SOURCES_REQ_ID_FK', 'sources', 'requests', ['request_id'], ['id'])
-        create_foreign_key('SOURCES_REPLICA_FK', 'sources', 'replicas', ['scope', 'name', 'rse_id'], ['scope', 'name', 'rse_id'])
+        create_foreign_key('SOURCES_REPLICA_FK', 'sources', 'replicas', ['rse_id', 'scope', 'name'], ['rse_id', 'scope', 'name'])
```

Comment:
Did you mean to reorder the foreign key columns here?

Reply:
This was the old wrong order [rse_id,scope,name] which got fixed by the upgrade 3345511706b8 to [scope,name,rse_id]. But in this upgrade it should be the old one.